hs_core/templates/pages/genericresource.html

```diff
@@ -779,8 +779,7 @@ <h4 class="modal-title">Confirm files deletion</h4>
                     {% if cm.resource_type != "CompositeResource" and cm|resource_type|lower != "collection resource" %}
                         <div class="col-xs-12 content-block">
                             <h3>Resource Specific</h3>
-                            <div class="{% if not is_resource_specific_tab_active %}tab-pane{% else %}tab-pane active{% endif %}"
```

Comment:
@sblack-usu This is where the context variable was used and it's no longer needed. This is the container of resource specific metadata.